clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -2267,6 +2268,14 @@ void Clang::AddSystemZTargetArgs(const ArgList &Args,
     CmdArgs.push_back("-mfloat-abi");
     CmdArgs.push_back("soft");
   }
+
+  if (Triple.isOSzOS()) {
+    CmdArgs.push_back("-mllvm");
+    CmdArgs.push_back(
+        Args.MakeArgString(llvm::Twine("-translation-time=")
+                               .concat(llvm::Twine(std::time(nullptr)))
+                               .str()));
+  }
```

Comment:
Is there a good reason for this to be a command-line option, or could this also be a module flag just like the others added above?  Seems odd to use two different mechanisms here.

Reply:
I refactored the translation time handling to use a module flag instead of a command line option.